pr-data.csv

```diff
@@ -4050,7 +4050,7 @@ https://github.com/fluent/fluent-logger-java,da14ec349bf0904da9865940b57b8756361
 https://github.com/fluent/fluent-logger-java,da14ec349bf0904da9865940b57b87563616ee04,.,org.fluentd.logger.TestFluentLogger.testReconnection,NDOD;UD;NOD,Unmaintained,,last commit on 2019-06-12
 https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-engine,net.redpipe.engine.swagger.SwaggerApiTest.checkSwagger,NOD,RepoArchived,,
 https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkMail,NOD,RepoArchived,,
-https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,Opened,https://github.com/fromage/redpipe/pull/56,
+https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,Skipped,https://github.com/FroMage/redpipe/commit/2fc605bc2fad4aadbb3b05282f528a05673df2ca#diff-86139da6bd888a5e7589f89d4f81a426bac99cd8aabab363efb5e338c1158f5bR131,
```

Comment:
Please leave `Opened` as it was and put `RepoArchived` in the last column (`Notes`).